src/lib/openapi/spec/context-field-schema.ts

```diff
@@ -5,41 +5,59 @@ export const contextFieldSchema = {
     $id: '#/components/schemas/contextFieldSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'A [context field](https://docs.getunleash.io/reference/unleash-context) used to configure strategies with data used to evaluate isEnabled calls',
     required: ['name'],
     properties: {
         name: {
+            description: 'The name of the context field',
             type: 'string',
+            example: 'userId',
         },
         description: {
+            description: 'The description of the context field.',
             type: 'string',
             nullable: true,
+            example: 'Used to uniquely identify users',
         },
         stickiness: {
+            description:
+                'Does this context field support being used for [stickiness](https://docs.getunleash.io/reference/stickiness) calculations',
             type: 'boolean',
+            example: true,
         },
         sortOrder: {
+            description:
+                'Used to have deterministic sorting of a list of context fields. Can also be used as a draw-breaker if list of context fields is sorted alphabetically.',
```

Comment:
I've never heard draw-breaker before, but I'm assuming it's the same as a tiebreaker? I'd suggest the latter because it seems to be more widely used.

What's the deterministic sort order you mention? What's it based on? 

```suggestion
                'Used when sorting a list of context fields. Is also used as a tiebreaker if a list of context fields is sorted alphabetically.',
```

Reply:
Good point, I don't know where drawbreaker came from either... :P